lms/djangoapps/student_account/views.py

```diff
@@ -421,6 +425,13 @@ def account_settings_context(request):
     user = request.user
 
     year_of_birth_options = [(unicode(year), unicode(year)) for year in UserProfile.VALID_YEARS]
+    try:
+        user_orders = get_user_orders(user)
+    except:  # pylint: disable=broad-except
```

Comment:
This is going to fail quality checks. You need to tell pylint to disable messages about "bare-except." See my previous example.